drivers/usbdev/cdcacm.c

```diff
@@ -2339,9 +2339,6 @@ static int cdcuart_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 
         /* And update with flags from this layer */
 
-        termiosp->c_iflag = serdev->tc_iflag;
-        termiosp->c_oflag = serdev->tc_oflag;
-        termiosp->c_lflag = serdev->tc_lflag;
```

Comment:
I am not sure I understand these three lines completely.  Is this because the values will be used in serial.c logic?

Reply:
These flags are saved in the TCSETS ioctl logic, but never referenced elsewhere.  Should they be removed from that location as well?  Should they be removed from the uart_dev_s structure?  Does anything use them?

Reply:
The code can be removed safely since the common code in serial.c already handle them correctly:
https://github.com/apache/nuttx/blob/master/drivers/serial/serial.c#L1556-L1601

Reply:
> These flags are saved in the TCSETS ioctl logic, but never referenced elsewhere. Should they be removed from that location as well? Should they be removed from the uart_dev_s structure? Does anything use them?

tc_iflag/tc_oflag/tc_lflag is used in serial.c to guard the various terminal special process.